TicTacToe_iOS/Views/ContentView.swift

```diff
@@ -28,39 +28,21 @@ struct ContentView: View {
                         Button(action: {
                             print("Top Left Button!")
                             game.TopLeft()
-                            noMove = game.notEmpty
```

Comment:
You can remove those variables from the top now since they are not necessary or required any more